### PR TITLE
Add support for a new (TSMgmtDataTypeGet) mgmt API to retrieve the record data type.

### PR DESCRIPTION
**Pull request by brbzull0**
Implements new management API that brings the record data type from a particular record. `TSMgmtDataTypeGet`

### CODE REVIEW

File: doc/developer-guide/api/functions/TSMgmtTypeGet.en.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+.. default-domain:: c
+
+TSMgmtTypeGet
+*************
+
+Synopsis
+========
+
+.. code-block:: cpp
```
Comment (SolidWallOfCode) [inline doc/developer-guide/api/functions/TSMgmtTypeGet.en.rst:27]: Isn't the API C and not C++?

**Comment by SolidWallOfCode** [comment]
Unfortunately, for plugin API changes you need to send mail to the dev list, with a title like "[API] TSMgmtRecordTypeGet". The body of the email can be the ".rst" file (well, most of it - leave out the boiler plate stuff).

**Comment by brbzull0** [comment]
> Unfortunately, for plugin API changes you need to send mail to the dev list, with a title like "[API] TSMgmtRecordTypeGet". The body of the email can be the ".rst" file (well, most of it - leave out the boiler plate stuff).

Will send an email to the list shortly.  Thanks.

**Comment by brbzull0** [comment]
[approve ci FreeBSD]

**Review by SolidWallOfCode — APPROVED** [review]

**Review by randall — APPROVED** [review]

**Pull request merged**